wordpress-mobile/WordPress-Android
# Viewers in People List

Closes #4312. This PR adds the Viewers to the dropdown in the people list page. Viewers is a WordPress.com only feature and only private blogs can have viewers. This feature follows the same patterns as the other filters, so should be mostly self-explanatory. I've also made some small improvements & fixed small bugs. Please let me know if something is not immediately clear, so I can improve the comments in the code to do so. Thanks!

**To test:** 
* Go into the People page for your private blog and select "Viewers" from the dropdown. If you have a viewer, it should show up there (if you don't have one, you should add it to test this feature).
* Go into the detail page for a viewer, check the UI and your details, then try and remove the viewer.
* Go into another blog that's not private and make sure the "Team" filter is selected since "Viewers" won't be available.

@mzorz I am asking @hypest to review this one just to make sure we get another set of eyes on it as it's very similar to the follower work. If you have the time to review it, I'd love to get your feedback as well!

/cc @astralbodies 

Needs review: @hypest
